### ISSUE TITLE
Identify "concepts" that could be "pulled out"

### ISSUE THREAD

**Issue by samm82** [JacquesCarette/Drasil#3698]
In #3686, I noticed that the phrase "wiki page", when used in `Sentence`s, often showed up alone, which makes me think it can be "pulled out" as an actual concept, such as those in `drasil-data`.

There are potentially many more similar concepts; they should be identified and listed, so we can have a conversation about which ones we want to define once and reuse (and how in-depth we want to be about this)!

As with #3252, this will likely be an "iterative" issue (i.e., it will never quite be "done"), so any amount of work on this is progress! We shouldn't worry about completeness just yet, or at least not on the first few passes!

**Comment by smiths** [comment]
Good catch @samm82.  I agree that we only need one issue on this topic.  Let's keep your issue and close the other one.
